lib/matplotlib/contour.py

```diff
@@ -144,6 +144,7 @@ def clabel(self, levels=None, *,
         # Detect if manual selection is desired and remove from argument list.
         self.labelManual = manual
         self.rightside_up = rightside_up
+        self.zorder = zorder
```

Comment:
make this private (otherwise, one could reasonably expect that setting it would update the zorder on the text object, which you probably don't want to bother doing)

Reply:
done.